Helper/Elasticsearch/HierarchicalStructure.php

```diff
@@ -59,6 +61,39 @@ public function setData($data): HierarchicalStructure
 
         return $this;
     }
+    
+    public function getActive(): bool
+    {
+        return $this->active;
+    }
+    
+    public function setActive(bool $active): HierarchicalStructure
+    {
+        $this->active = $active;
+
+        return $this;
+    }
+    /**
+     * Search all active items for a given Id
+     * @return HierarchicalStructure
+     */
+    public function setAllActives(string $currentId): HierarchicalStructure
+    {   
```

Comment:
This function can be removed if we use the recursion of getHierarchy

Reply:
done